src/plays/hangman-game/constants/constants.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export const KEYS = [
+  'a',
+  'b',
+  'c',
+  'd',
+  'e',
+  'f',
+  'g',
+  'h',
+  'i',
+  'j',
+  'k',
+  'l',
+  'm',
+  'n',
+  'o',
+  'p',
+  'q',
+  'r',
+  's',
+  't',
+  'u',
+  'v',
+  'w',
+  'x',
+  'y',
+  'z'
+];
+
+export const tryAgainIcon = (
```

Comment:
name could be changed to `TRY_AGAIN_ICON`

Reply:
@sudipkundu999  done!